run_register_rln.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+
+if test -f ./keystore/keystore.json; then
+  echo "keystore/keystore.json already exists. Use it instead of creating a new one."
+  echo "Exiting"
+  exit 1
+fi
+
```

Comment:
can we merge this script with the `run_nwaku.sh`?

Reply:
believe this was addressed in 98cfe98ae2c72e17def3b91e3812babb277e659f ?
@stubbsta

Reply:
yes, resolved